.ddev/.env.template

```
@@ -77,3 +77,10 @@ ZMS_SOURCE_DLDB_BERLIN=https://service.berlin.de
 ZMS_SOURCE_DLDB_MUNICH=https://stadt.muenchen.de/service/info/zms/index/
 # Legacy default - uses Berlin format
 ZMS_SOURCE_DLDB=https://service.berlin.de
+
+# zmscitizenapi: if HTTP_HOST or X-Forwarded-Host contains this substring, unpublished
+# services/relations are still returned (e.g. Munich SADB ZMS_INTERN=true → public=false after import).
+# One substring per env; comma-separated lists are not supported by the app.
+# Do not use zms-web here unless you never call the citizen API on your public ddev URL (it would stop matching).
+# refarch-gateway citizen-api uses PreserveHostHeader so the browser host is forwarded to PHP.
+ACCESS_UNPUBLISHED_ON_DOMAIN=localhost
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)